tensorflow/core/kernels/mlir_generated/base_ops_test.h

```diff
@@ -199,6 +199,16 @@ absl::InlinedVector<T, 10> DefaultInputNonZero() {
                                          0.2, 0.3, 0.5, 0.7, 0.9, 9.0, 18.0});
 }
 
+template <typename T, std::enable_if_t<
+                          llvm::is_one_of<T, Eigen::half, float, double>::value,
+                          bool> = true>
+absl::InlinedVector<T, 10> VeryLargeVector() {
+  auto max_val = pow(2,32) + 1;
```

Comment:
I am not sure whether we want to have a test that has 2^32 + 1 many elements. Do we really need an end-to-end test for this? I also see that you have currently commented out the test that uses this function.

@frgossen what do you think?

Reply:
Yes, we have codegen tests. I think, the e2e test should exist somewhere. We can always disable it if it is too expensive.